Tensile/KernelWriterAssembly.py

```diff
@@ -7175,6 +7175,18 @@ def mfmaIter(self, kernel, u, innerUnroll, vregSetIdx, lastKinloop=False, tail=F
     # release register
     if abReg is not None: self.vgprPool.checkIn(abReg)
 
+    # conditions to reverse idxInner if idxOuter is odd (for better HW efficiency)
+    # disable reverse idxInner if one of the following condition is true
+    # - PGR=2 and u == LoopIters - 1 and DTV
+    # - wmma
+    # - complex type
+    enableReverseInner = True
```

Comment:
Is it worth having a parameter to turn this off? Or is this mode always faster?

Reply:
I would not like to add a new parameter for this.
This makes only small difference.
My current assumption is that it is not worth trying both.

Reply:
Since this is more power efficient according to HW team, I think it is ok to always enable this (as long as it is applicable).